src/NetTopologySuite.IO.Oracle/Sdo/SdoEType.cs

```diff
@@ -8,6 +8,7 @@ internal enum SdoEType
         Coordinate = 1,
         Line = 2,
         Polygon = 3,
+        Multipoint = 4,
```

Comment:
Where did you find that definition? I can't find it in [Oracle's definition of SDO_Geometry](https://docs.oracle.com/database/121/SPATL/sdo_geometry-object-type.htm#GUID-270AE39D-7B83-46D0-9DD6-E5D99C045021__BGHDGCCE)

Reply:
Ohh... It's my mistake - I didn't realized that the plan was that those values should be mapped to the SDO_ETYPE if so than problem is a bit more complex... [SDO_GTYPE](https://docs.oracle.com/en/database/oracle/oracle-database/23/spatl/sdo_geometry-object-type.html#GUID-4D84CE67-67E4-4A84-8255-C586E765A94B) contains different values for point and multipoint, but SDO_ETYPE has the same value, but different SDO_INTERPRETATION if geometry is a multipoint. Thanks for your comment - will be back with proper solution, also will add tests and verify if same issue occurs on multi lines and multi polygons

Reply:
@FObermaier seams that it is oracle limitation and in oracle you can't create geometry collection with multipoint which contains only one point... while inserting you will loose multipoint type... so that change has been reverted, but still there is (was) issue with saving Multipoint with one point, and changes in OracleGeometryReader are required

Reply:
Yes I have access to an ORACLE database and I've spent some time with playing with that - and actually you can create GeometryCollection which contains a MultiPoint and it works fine if you have more than one points in MultiPoint. If you have only one point unfortunately you lose information that it was MultiPoint. 

Let me describe it a bit more:
For geometry collection SDO_GTYPE is DL04 (eg. 2004) to specify what geometries are inside you must define SDO_ELEM_INFO, which contains SDO_STARTING_OFFSET , SDO_ETYPE and SDO_INTERPRETATION. As offset is obvious we can "ignore it". ETYPE for Point and Multipoint are the same (it's eq to 1) only difference maybe in the interpretation - which is equal 1 for point and "n > 1" for "Point cluster with n points.". Issue is - what happens if Point cluster contain only 1 point - then elem_info for multipoint and point are exact the same.

I did my best to insert "GEOMETRYCOLLECTION(MULTIPOINT(11 12))" into oracle database (I did it manually and via external tools) it looks exact the same to "GEOMETRYCOLLECTION(POINT(11 12))".

Anyway, fixed that I did in this PR solves issue #15 - you can save and read "MULTIPOINT(11 12)"

Reply:
> I did my best to insert "GEOMETRYCOLLECTION(MULTIPOINT(11 12))" into oracle database (I did it manually and via external tools) it looks exact the same to "GEOMETRYCOLLECTION(POINT(11 12))".

That is what I wanted to make sure.